temporal-sdk/src/main/java/io/temporal/internal/external/GenericWorkflowClientExternalImpl.java

```diff
@@ -78,7 +79,9 @@ public WorkflowExecution start(StartWorkflowExecutionRequest request) {
     StartWorkflowExecutionResponse result;
     result =
         GrpcRetryer.retryWithResult(
-            GrpcRetryer.DEFAULT_SERVICE_OPERATION_RETRY_OPTIONS,
+            RpcRetryOptions.newBuilder()
```

Comment:
Nit: Might be nice to have a little convenience function on `RpcRetryOptions` that does this three line combo being repeated a bunch in these diffs.

Reply:
Yeah, we could add something like `buildWithDefaultsFrom`.